app/views/candidate_interface/course_choices/shared/_go_to_find.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.find_a_course') %>
+    </h1>
+
+    <p class="govuk-body-l">Search for courses near you on Find postgraduate teacher&nbsp;training.</p>
+
+    <%= link_to 'https://find-postgraduate-teacher-training.education.gov.uk', role: 'button', class: 'govuk-button govuk-!-margin-top-4 govuk-!-margin-bottom-2 govuk-button--start', target: '_blank', rel: 'noopener', 'data-module': 'govuk-button', 'aria-describedby': 'find-new-window' do %>
+      <%= t('application_form.begin_button') %>
+      <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" role="presentation" focusable="false">
+        <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z" />
```

Comment:
We've got this 7 times in the codebase, we should create a component at some point!

Reply:
cool. I'll create another PR for it 👍